src/Data/heroData.json

```diff
@@ -0,0 +1,22 @@
+{
+    "header": "I'm just a header",
+    "description": "And i'm a simple description",
+    "cta": {
+      "label": "Click me please (: ",
+      "url": "www.hiii.com"
+    },
+    "images": {
+      "small": {
+        "url": "https://wallpapercave.com/wp/Fe3XMh0.jpg",
+        "description": "This is the small image description"
+      },
+      "medium": {
+        "url": "http://i.imgur.com/SkH0dnQ.jpg",
+        "description": "This is the midium image description"
+      },
+      "large": {
+        "url": "http://markinternational.info/data/out/768/225789461-nature-wallpaper-1920x1080.jpg",
+        "description": "This is the large image description"
+      }
+    }
+  }
```

Comment:
You shouldn't have a file just for the hero data. You should have a file that contains the data for a complete page.